cobigen-cli/src/main/java/com/devonfw/cobigen/cli/utils/CobiGenUtils.java

```diff
@@ -185,7 +185,7 @@ public CobiGen initializeCobiGen() {
         CobiGen cg = null;
         if (templatesJar != null) {
             try {
-                registerPlugins();
+            	registerPlugins();  
```

Comment:
Here you have added some extra tabs that we don't need, please remove those whitespaces